package.json

```diff
@@ -2,12 +2,14 @@
   "private": true,
   "dependencies": {
     "koa": "^2.11.0",
-    "koa-logger": "^3.2.1"
+    "koa-logger": "^3.2.1",
+    "koa-router": "^7.4.0"
```

Comment:
Looks like this is abandoned and https://github.com/koajs/router should be used instead.